docs/crud.md

```diff
@@ -0,0 +1,103 @@
+# Access a document
+
+#### Request
+```http
+GET /data/:type/:id
+```
+```http
+GET /data/types.cozy.io/events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee
+```
+
+#### Response OK
+```http
+HTTP/1.1 200 OK
+Date: Mon, 27 Sept 2016 12:28:53 GMT
+Content-Length: ...
+Content-Type: application/json
+Etag: "3-6494e0ac6494e0ac"
+```
+```json
+{
+    "_id": "types.cozy.io/events/6494e0ac-dfcb-11e5-88c1-472e84a9cbee",
+    "_rev": "3-6494e0ac6494e0ac",
+    "_type": "types.cozy.io/events",
+    "startdate": "20160823T150000Z",
+    "enddate": "20160923T160000Z",
+    "summary": "A long month",
+    "description": "I could go on and on and on ....",
+}
+```
+
+### Response Error
+```http
+HTTP/1.1 404 Not Found
+Content-Length: ...
+Content-Type: application/json
+```
+```json
+{
+  "status": 404,
+  "error": "not_found",
+  "reason": "deleted",
```

Comment:
This is linked to couchdb : deleting a doc is equivalent to create a new tombstone revision ( `{"_id": "types.cozy.io/events/8239239283", "_rev":"5-72372382873", "_deleted": true}` ). This is necessary to ensure proper replication.

Former revisions of the document are also kept by couchdb, but will be removed by a **compact** operation when disk space starts to become a problem. However, the tombstone will remain.
Removing tombstone is done with the **purge** operation, this is rarely necessary as the tombstone weight only a few 10s bytes.